packages-exp/auth-exp/src/index.ts

```diff
@@ -54,3 +54,6 @@ export { reload } from './core/user/reload';
 
 // model
 export { Operation as ActionCodeOperationType } from './model/action_code_info';
+
+// platform-browser/recaptcha
+export { RecaptchaVerifier } from './platform_browser/recaptcha/recaptcha_verifier';
```

Comment:
@avolkovi wdyt of putting this here? I *think* it's okay since if you don't use it it will be tree-shaken out

Reply:
seems fine, bassam's api proposal has

import {
 browserLocalPersistence,
 RecaptchaVerifier,
 signInWithPhoneNumber, // Depends on PhoneAuthProvider.
 initializeAuth,
} from 'firebase/auth/core';